bodhi/server/models.py

```diff
@@ -4141,6 +4141,7 @@ class Comment(Base):
     Attributes:
         karma (int): The karma associated with this comment. Defaults to 0.
         karma_critpath (int): The critpath karma associated with this comment. Defaults to 0.
+            **DEPRECATED** no longer used in the UI
```

Comment:
This is going to go in as is but I was wondering if we should link to the ticket here as a reason why it's deprecated but not removed (we can address this in another PR though).